user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures.md

```diff
@@ -595,3 +595,19 @@ To change this frequency:
 1. Specify the new value and click *OK*.
 
 1. Click *OK* to close the *SLNet Options* window.
+
+## Disabling automatic NATS configuration
+
+From DataMiner 10.2.0 \[CU6]/10.2.8, you can enable the *NATSForceManualConfig* option so that NATS is not automatically configured in your DataMiner System. When you do so, you will need to either configure a NATS cluster manually instead, or manually call the *NatsCustodianResetNatsMessage* when changes are made to the DMS (see [Try a NATS reset](xref:Investigating_NATS_Issues#try-a-nats-reset)).
+
+To disable automatic NATS configuration:
+
+1. Go to *Advanced* > *Options* > *SLNet Options*.
+
+1. In the drop-down box, select *NATSForceManualConfig*.
+
+1. Right-click the DMA for which you want to change the setting and select *Edit value*.
```

Comment:
The option should have the same value for every DMA in the cluster, otherwise, the one that does not have ForceManualConfig will still run the reset timer and could call NatsCustodianResetNatsMessage, which could destroy the manually set up NATS configuration.